draft-ietf-quic-invariants.md

```diff
@@ -150,10 +150,25 @@ version-specific semantics are marked with an X.
 {: #fig-long title="QUIC Long Header"}
 
 A QUIC packet with a long header has the high bit of the first octet set to 1.
+All other bits in that octet are version specific.
+
+The next octet contains the length in octets of the two Connection IDs (see
+{{connection-id}}) that follow.  Each length is encoded as a 4 bit unsigned
+integer.  The length of the Destination Connection ID occupies the high bits of
+the octet and the length of the Source Connection ID occupying the low bits of
+the octet.  An encoded length of 0 indicates that the connection ID is also 0
+octets in length.  Non-zero encoded lengths are increased by 7 to get the full
+length of the connection ID.  For example, an octet with the value 0xa0
+describes a 17 octet Destination Connection ID and a zero octet Source
+Connection ID.
 
-A QUIC packet with a long header has two fixed fields immediately following the
-first octet: a 64-bit Connection ID (see {{connection-id}}) and a 32-bit Version
-(see {{version}}).
+The connection ID lengths are followed by connection IDs.  The connection ID
+selected by the recipient of the packet (the Destination Connection ID) is
```

Comment:
This isn't quite correct for the initial Initial packet. Maybe "associated with the recipient"